code/modules/research/rdconsole.dm

```diff
@@ -170,7 +170,7 @@ doesn't have toxins access.
 					var/obj/item/card/id/ID = I.GetID()
 					if(istype(ID))
 						logname = "User: [ID.registered_name]"
-			stored_research.research_logs += "[logname] researched node id [id] for [price] points."
+			stored_research.research_logs += "[logname] researched node id [id] with cost [json_encode(price)]."

```

Comment:
I get what you're going for but using `json_encode` for this is a little goofy, maybe change it to format nicer when you actually go to use this system.

Reply:
@SpaceManiac if anything the input() strings are uglier, logs are generally not eye-candy-ish when viewed raw, but if you want i can try to make it better ? research_logs isn't really used rn

Reply:
Not a big deal, just keep it in mind if you actually use this system.